examples/gke/gke_cluster_gitops/main.tf

```diff
@@ -0,0 +1,32 @@
+resource "castai_gke_cluster" "my_castai_cluster" {
```

Comment:
```suggestion
resource "castai_gke_cluster" "this" {
```

We will have only one resource with that name :)

Reply:
Valid point, thanks! 🚀